### PR TITLE
CON-2800 Markdown `Quest00/Ex00`

### PR DESCRIPTION
**Pull request by Oumaimafisaoui**
[CON-2800](https://01talent.atlassian.net/browse/CON-2800)

[CON-2800]: https://01talent.atlassian.net/browse/CON-2800?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: subjects/DPxAI/skeleton/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+- Video [Where to start](https://www.youtube.com/watch?v=i2PWvTERU-g&list=PLDa5D3mQAy7SKRLbYIEAcPrbQxrR_g1uM&index=1)
+- Video [Basic set up of an HTML page](https://www.youtube.com/watch?v=QtKoO7tT-Gg&list=PLHyAJ_GrRtf979iZZ1N3qYMfsPj9PCCrF&index=1)
+- Video [Different HTML tags overview](https://www.youtube.com/watch?v=Al-Jzpib8VY&list=PLHyAJ_GrRtf979iZZ1N3qYMfsPj9PCCrF&index=2)
+
+Those videos are accompanying you step by step in each exercise, but if you want to check right away all the notions covered in the quest, you can watch the whole playlist [Web - HTML, CSS & DOM JS](https://www.youtube.com/playlist?list=PLHyAJ_GrRtf979iZZ1N3qYMfsPj9PCCrF).
+
+> Your working environment may not be exactly the same as what you see in the videos or documentation, just try to adapt your work according to your research and discoveries.  
+> Don't be afraid to try!
+
+### Instructions
```
Comment (nprimo) [inline subjects/DPxAI/skeleton/README.md:33]: From the instructions I would expect the files to be delivered to be an `.html` file, while in the current ref is `skeleton.js`.

```json
# 20101.json
{
  "type": "exercise",
  "name": "skeleton",
  "attrs": {
    "displayedName": "Skeleton",
    "expectedFiles": ["skeleton.js"],
    "language": "dom",
    "validations": [
      {
        "type": "tester",
        "testImage": "ghcr.io/01-edu/test-dom",
        "cooldown": 0
      }
    ]
  }
}
```

**Review by nprimo — CHANGES_REQUESTED** [review]

File: subjects/DPxAI/.DS_Store
[the remark this answers is not in the document]
Comment (Oumaimafisaoui) [reply subjects/DPxAI/.DS_Store:1]: Fixed an issue in .gitignore file.

File: subjects/DPxAI/skeleton/.DS_Store
[the remark this answers is not in the document]
Comment (Oumaimafisaoui) [reply subjects/DPxAI/skeleton/.DS_Store:1]: global ```.gitignore``` file is modified.

File: subjects/DPxAI/skeleton/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+> Brainpower mode
+
+### Context
+
+To craft your robot friend, you first need to build its skeleton, and that's where HTML comes in. HTML is the blueprint that sets up the structure of your robot, giving it shape and form. With HTML, you'll create the framework that holds all your amazing ideas together. Ready to start building? Let’s dive in and create the foundation of your robot buddy!
+
+> For the whole quest, the principle is to iterate over your code: when you finish an exercise, copy-paste your code to use it in the next one.
+
+> Contrary to what you will see in the videos, you don't have to run a server on your machine to use the html css javascript thanks to the integrated code editor that takes care of all that.
```
[the remark this answers is not in the document]
Comment (Oumaimafisaoui) [reply subjects/DPxAI/skeleton/README.md]: Fixed

File: subjects/DPxAI/skeleton/README.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+> Don't be afraid to try!
+
+### Instructions
+
+Ready? Let's code!
+
+> The first step to achieve in your quest is to craft the shape of your robot ; for that, 2 things needs to be done:
+>
+> - Custumize the( [`<head>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head) tag)
+> - Custumize the ([`<title>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/title) tag)
```
[the remark this answers is not in the document]
Comment (Oumaimafisaoui) [reply subjects/DPxAI/skeleton/README.md]: Fixed.

[the remark this answers is not in the document]
Comment (Oumaimafisaoui) [reply subjects/DPxAI/skeleton/README.md]: Fixed.

File: subjects/DPxAI/skeleton/README.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+- super-strength
+- advanced-healing
+- mind-link
+
+#### The `<body>` part:
+
+This section contains the content of your webpage , in our case the lower body of your buddy. Define the skeleton of your robot by organizing it into three main sections: the face, the upper body, and the lower body. Inside the <body> tag of your HTML file, create three divisions using <section> tags, and place the following text content inside each one: face, upper-body, lower-body.
+
+> Don't forget to press Render again to refresh the display page when you modify your code.  
+> You can also see javascript code in the console at the bottom of the page, as in the previous quest.
```
[the remark this answers is not in the document]
Comment (Oumaimafisaoui) [reply subjects/DPxAI/skeleton/README.md]: Fixed

File: subjects/DPxAI/skeleton/README.md (hunk 1 above)
Comment (Oumaimafisaoui) [reply subjects/DPxAI/skeleton/README.md:32]: The refs will be recreated for the new piscine.

**Review by nprimo — APPROVED** [review]
LGTM

**Pull request merged**